project/src/main/java/com/google/sps/utilities/DatabaseWrapper.java

```diff
@@ -20,21 +20,15 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+/** Class containing methods for interaction with database. */
 public class DatabaseWrapper {
-  private String instanceId;
-  private String databaseId;
+  private final DatabaseService databaseService;
   private static final String USER_TABLE = "Users";
   private static final String VOLUNTEERING_OPPORTUNITY_TABLE = "VolunteeringOpportunity";
   private static final String EVENT_TABLE = "Events";
-  //TO DO: replace with db injection on PR #57
-  private static final SpannerOptions options = SpannerOptions.newBuilder().build();
-  private static final Spanner spanner = options.getService();
-  private final DatabaseId db = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
-  private final DatabaseClient dbClient = spanner.getDatabaseClient(db);
 
-  public DatabaseWrapper(String instanceId, String databaseId) {
-    this.instanceId = instanceId;
-    this.databaseId = databaseId;
+  public DatabaseWrapper(DatabaseService databaseService) {
+    this.databaseService = databaseService;
```

Comment:
why not also have a private final databaseClient = databaserService.getDatabaseClient() here?

Reply:
I think we cannot because this is called only when the servlet is initialized (runs only once). So if we initialize using private static final and close it in doGet(), then the next time we do a GET, the connection that came from when the servlet is initialized will be closed.

Reply:
Ack - I can change it, I was using this example for dependency injection where they call owner.getPet() in the main function. I was thinking owner is like DatabaseWrapper and Pet is like databaseService. https://docs.google.com/document/d/1gAf3i91d1PSmNr329BziJbNBc81JbfxdhXYnSivjxNg/edit#heading=h.ny5w2jug54cu